blackjack/interface.py

```diff
@@ -0,0 +1,101 @@
+from player import Player
+from dealer import Dealer
+import sys
+
+class Interface:
+    """The prompts for displaying of input & output
+
+    Responsibilities:
+
+    * Ask Player for input
+    * Displays output messages
+
+    Collaborators:
+
+    * Works with Game class
+    *"""
```

Comment:
Love this! Having a separate Interface class is absolutely great. The only comment I'd make is that since this doesn't actually contain any data, you could have just made it a set of functions in a module. Still, this is great.